File: week-1/sorted array check/sorted_array_check.c
// Check if array is sorted

#include<stdio.h>
int main() {

    int n,flag = 1;

    printf("Enter number of elements : ");
    scanf("%d",&n);

    int arr[n];


    printf("Enter %d elements : \n",n);
    for(int i=0; i<n; i++) {
        scanf("%d",&arr[i]);
    }

    for(int i=0; i<n-1; i++) {
        if(arr[i] > arr[i+1]) {
            flag = 0;
            break;
        }
    }


    if(flag == 1) {
        printf("Array is sorted in ascending order. ");
    }
    else{
        printf("Array is not sorted in ascending order. ");
    }

    
    return 0;
}

File: week-1/sorted array check/sorted_array_check.py
arr = list(map(int, input("Enter array elements separated by space : ").split()))

is_sorted = True
for i in range(len(arr)-1):
    if arr[i] > arr[i+1] :
        is_sorted = False
        break


if is_sorted:
    print("Array is sorted in ascending order .")
else:
    print("Array is not sorted in ascending order .")
